src/families/tron/AccountBalanceSummaryFooter.js

```diff
@@ -150,4 +159,41 @@ const styles = StyleSheet.create({
   },
 });
 
-export default AccountBalanceSummaryFooter;
+function getInfoCandidates(t: TFunction): { [key: InfoName]: ModalInfo[] } {
+  const currency: CryptoCurrency = (findCryptoCurrencyById("tron"): any);
+  const TronIcon: React$ComponentType<{
+    color: string,
+    size: number,
+  }> = (getCryptoCurrencyIcon(currency): any);
+
+  return {
+    available: [
+      {
+        Icon: () => <TronIcon color={currency.color} size={18} />,
+        title: t("tron.info.available.title"),
+        description: t("tron.info.available.description"),
+      },
+    ],
+    frozen: [
+      {
+        Icon: () => <FreezeIcon size={18} />,
+        title: t("tron.info.frozen.title"),
+        description: t("tron.info.frozen.description"),
+      },
+    ],
+    bandwidth: [
+      {
+        Icon: () => <BandwidthIcon size={18} />,
+        title: t("tron.info.bandwidth.title"),
+        description: t("tron.info.bandwidth.description"),
+      },
+    ],
+    energy: [
+      {
+        Icon: () => <EnergyIcon size={18} />,
+        title: t("tron.info.energy.title"),
+        description: t("tron.info.energy.description"),
```

Comment:
it sounds like these could all be statically defined if you were using `<Trans`

Reply:
Stick with good old t func that simply returns string for now.